### ISSUE TITLE
Tab-Order

### ISSUE THREAD

**Issue by m0ru** [researchstudio-sat/webofneeds#1102]
Atm there's (almost) no elements with a tab-index set. At the very least we should set a `tabindex="0"` on the important navigation and interface elements.

**Comment by fkleedorfer** [comment]
@m0ru done?

**Comment by m0ru** [comment]
Nope. The first step would be to auto-focus the textareas in create-need and an open chat. But we really should have tab-indizes for the entire thing.

We also don't have alt-texts and aria-roles. Switching to elm might help us here, as will ensure that we can't forget to assign these.

**Comment by peacememories** [comment]
Is there still anything needing clarification here? This might be ready to be moved to "Soon" or "Later"
